File: php/Processor.php
<?php
   
    class Processor{
        //aatribut
        private $name = "";
        private $price = 0;

        //konsruktor
        function __construct($name = "", $price = 0){
            $this->name = $name;
            $this->price = $price;
        }

        //setter & getter
        function setname($name){
            $this->name = $name;
        }
        function getname(){
            return $this->name;
        }

        function setprice($price){
            $this->price = $price;
        }
        function getprice(){
            return $this->price;
        }

        function printProcessor(){
            echo "Processor   : ".$this->getname()."<br/>";
        }

        //destruktor
        function __destruct(){  
        }
    } 
?>

File: python/processor.py
class Processor():
    #private members
    __price = 0
    __name = "blank"

    def __init__(self, name = "blank", price = 0):
        self.__price = price
        self.__name = name

    #getter & setter
    def setprice(self, price):
        self.__price = price

    def getprice(self):
        return self.__price
    
    def setname(self, name):
        self.__name = name

    def getname(self):
        return self.__name

    def printProcessor(self):
        print( "Processor   : %s" %(self.__name))
